### DISCUSSION TITLE
[Vote ended on 2022-12-08] Remove cisco.nso from Ansible 9

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#165] [Votes done]
`cisco.nso` looks unmaintained (#155).

Please vote with a +1 to remove it from Ansible 9 or -1 to keep it in this discussion.

**Comment by mariolenz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by gotmax23** [comment]
+1 for the reasons listed in https://github.com/ansible-community/community-topics/issues/155#issuecomment-1324572267.

**Comment by russoz** [comment]
+1

**Comment by gundalow** [comment]
+1
